docs/guide/reactive-progressive.md

```diff
@@ -0,0 +1,205 @@
+# Use Status Management Progressive
+
+> If you want to use whole Goldfish Status Management, please visit [Use Status Management](./reactive.md).
+
+## Alipay mini program status management
+
+The original framework of Alipay mini program provides the page/component level data status management ([this.setData()](https://docs.alipay.com/mini/framework/page-detail#pageprototypesetdatadata-object-callback-function)), but that is not available for page or component across scenarios. The general solution is to put data into globalData to solve the data sharing problem across pages.
+
+It can be seen that basic mechanism is available for status management in the original Alipay mini program, which does not suffice the status management in complicated projects.
+
+## Goldfish responsive status management
+
+On basis of the related great ideas in the community, Goldfish deposits a set of mini program friendly responsive status management framework, which enables the user to flexibly and efficiently handle the status data.
+
+In Goldfish, status data is responsive, you can use `@goldfishjs/reactive-connect` declare responsive data, and use `AppStore` `PageStore` `ComponentStore` declare store, finally use `createApp` `createPage` `createComponent` to connect store data.
```

Comment:
you can use `observable`, `state`, `computed` to create reactive data.

Reply:
done